make/buf/all.mk

```diff
@@ -27,7 +27,9 @@ FILE_IGNORES := $(FILE_IGNORES) \
 	internal/buf/cmd/buf/cache/ \
 	internal/buf/cmd/buf/workspacetests/other/proto/workspacetest/cache/ \
 	internal/buf/internal/buftesting/cache/ \
-	internal/pkg/storage/storageos/tmp/
+	internal/pkg/storage/storageos/tmp/ \
+	**/crashers/*.output \
+    **/crashers/*.quoted
```

Comment:
Nit: I think this is mixing tabs and spaces

Reply:
Good eye. Spaces in a Makefile is a classic blunder.